MTV-5065: Release Notes 2.11.4 (#900)
kubev2v/forklift-documentation: documentation/modules/new-features-and-enhancements-2-11.adoc

              policy: optional
            - name: svm
              policy: optional
----
====
+

> link:https://issues.redhat.com/browse/MTV-4698[(MTV-4698)]

Reviewer: ```suggestion
link:https://issues.redhat.com/browse/MTV-4698[MTV-4698]
```

Reviewer: Removed the brackets here like all the other links in this file.

+
link:https://issues.redhat.com/browse/MTV-3130[MTV-3130]

> Enabled latest virtio-win drivers for improved Azure performance::

Reviewer: Can we get any clarity on the Azure reference in the heading because it's not the main text? Is the enhancement specific to Azure or does it apply to Windows VMs in general?

Are we not adding release notes directly to the Jira fields?

Reviewer 2: The azure reference should not be there IMO. It's not related to Azure but general improvements

Reviewer: ```suggestion
Updated virtio-win drivers for improved migration stability::
```

> The `virtio-win` package, which provides drivers for Microsoft Windows virtual machines, is rebased to version 1.9.57. With this update, virtual machine migrations are more stable and include security updates. This update also resolves rare instances of Blue Screen of Death (BSOD) errors that occurred during migrations.

Author: ```suggestion
The `virtio-win` package, which provides drivers for Microsoft Windows virtual machines, is rebased to version 1.9.57. With this update, virtual machine migrations are more stable and include security updates. This update also resolves rare instances of stop errors that occurred during migrations.
```

+

> link:https://redhat.atlassian.net/browse/MTV-5077[MTV-5077]

Author: ```suggestion
// link:https://redhat.atlassian.net/browse/MTV-5077[MTV-5077]
```

> link:https://redhat.atlassian.net/browse/MTV-5077[MTV-5077]

Reviewer 2: MTV-5077 is private for now, I will make it public once we going to release.

> With this update, the integration of the latest virtio-win-1.9.57-0.el9_6 drivers from Red Hat is enabled, streamlining the driver installation process for improved system performance and compatibility within Microsoft Azure clusters. This proactive step ensures smooth and stable performance of virtual Windows environments from April 21st, providing better security, compatibility with the latest Windows operating systems, and potential performance enhancements for end users.

Reviewer 2: Not sure where Azure clusters come from, but I think we should say something along these lines:

With the driver update, the migrations now offer more stable performance, security updates and resolve multiple(? not sure if this would be the right word... perhaps "rare"?) cases of BSOD during migrations.

> With this update, the integration of the latest virtio-win-1.9.57-0.el9_6 drivers from Red Hat is enabled, streamlining the driver installation process for improved system performance and compatibility within Microsoft Azure clusters. This proactive step ensures smooth and stable performance of virtual Windows environments from April 21st, providing better security, compatibility with the latest Windows operating systems, and potential performance enhancements for end users.

Author: ```suggestion
The `virtio-win` package, which provides drivers for Microsoft Windows virtual machines, is rebased to version 1.9.57. With this update, virtual machine migrations are more stable and include security updates. This update also resolves rare instances of Blue Screen of Death (BSOD) errors that occurred during migrations.
```
